olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -171,6 +171,17 @@ bool DiskCache::Clear() {
   return true;
 }
 
+bool DiskCache::CheckState() {
+  if (database_) {
+    std::string property_result;
+    database_->GetProperty("leveldb.num-files-at-level0", &property_result);
+
+    const auto files_at_level0 = std::stoi(property_result);
+    return files_at_level0 < 4;
```

Comment:
No magic numbers, please. Add a constexpr in the anonymous namespace.

Reply:
done